### ISSUE TITLE
Not showing all recordings

### ISSUE THREAD

**Issue by johnkdevnull** [MythTV-Clients/MythTV-Android-Frontend#94]
I currently have episode from three series in my recorded list and also one film.  The film does not show in the App Recordings list.  Is it only showing items that have episode info?

Using
MythTV Version : 0.26.0-1.fc17 (v0.26.0-28-ge3087dd)  
MythTV Branch : fixes/0.26  
Network Protocol : 75  
Library API : 0.26.20120822-1
QT Version : 4.8.2
Options compiled in:
 linux release use_hidesyms using_alsa using_jack using_oss using_pulse using_pulseoutput using_backend using_bindings_perl using_bindings_python using_bindings_php using_crystalhd using_dvb using_firewire using_frontend using_hdhomerun using_ceton using_hdpvr using_iptv using_ivtv using_joystick_menu using_libcrypto using_libdns_sd using_libfftw3 using_libxml2 using_libudf using_lirc using_mheg using_opengl_video using_qtwebkit using_qtscript using_qtdbus using_v4l2 using_x11 using_xrandr using_xv using_bindings_perl using_bindings_python using_bindings_php using_mythtranscode using_opengl using_vaapi using_vdpau using_ffmpeg_threads using_live using_mheg using_libass using_libxml2 using_libudf

**Comment by billmeek** [comment]
Try browsing the following:

```
http://<yourBackendIPv4Address>:6544/Dvr/GetRecordedList
```

Using your browser's search feature, do you see the "one film" you're looking for?

The line would look similar to this:

```
<Title>The Title of Your Missing Film Here</Title>
```

The goal of this test is to see if MythTV's Services API is returning the information.

**Comment by johnkdevnull** [comment]
On 12/18/2012 10:36 PM, billmeek wrote:

> Try browsing the following:
> 
> |http://<yourBackendIPv4Address>:6544/Dvr/GetRecordedList
> |
> 
> Using your browser's search feature, do you see the "one film" you're 
> looking for?
> 
> The line would look similar to this:
> 
> |<Title>The Title of Your Missing Film Here</Title>
> |
> 
> The goal of this test is to see if MythTV's Services API is returning 
> the information.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/MythTV-Clients/MythTV-Android-Frontend/issues/94#issuecomment-11508754. 
> 
> Thanks for responding, yes the listing does include the film (The 
> A-Team).  See the attached file.

Kind regards

JohnK

**Comment by billmeek** [comment]
@johnkdevnull, Hi; the attachment didn't make it to email or to the issues list on github.

If you could save the output from your browser and use pastebinit <someFileName>
then put the link here, that would be perfect. (I think you're running fedora so maybe
that would be yum install pastebinit if you don't already have it.) Thanks!

For what it's worth, I did have a movie (the expire process removed it on the backend)
and it showed up on my tablet.

**Comment by johnkdevnull** [comment]
Here it is

http://fpaste.org/WOIg/

Hope you can read that.  I just catted the file into pastebinit.

**Comment by billmeek** [comment]
@johnkdevnull Thanks. I forgot that the saved file is one line. 

Here's a reformatted <program> ... </program> section for the movie in question:

http://paste.ubuntu.com/1452741/

@dmfrey Note that the A-Team movie is in the LiveTV storage group.

**Comment by dmfrey** [comment]
@johnkdevnull I believe the issue is related to the fact that we are
removing articles from titles so as to group them much like a traditional
FE does.  So in this case, it might be that 'the' and 'A' are being
removed.  Do you have a program group called 'Team' in your interface?

On Thu, Dec 20, 2012 at 12:05 PM, billmeek notifications@github.com wrote:

> @johnkdevnull https://github.com/johnkdevnull Thanks. I forgot that the
> saved file is one line.
> 
> Here's a reformatted ... section for the movie in question:
> 
> http://paste.ubuntu.com/1452741/
> 
> @dmfrey https://github.com/dmfrey Note that the A-Team movie is in the
> LiveTV storage group.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/MythTV-Clients/MythTV-Android-Frontend/issues/94#issuecomment-11581545.

## 

My Google Profile http://www.google.com/profiles/dmfrey

**Comment by johnkdevnull** [comment]
@dmfrey I don't have a program group called Team.

It shows in my recorded list in Mythfrontend.  I'd expect it to show in the Android frontend.

**Comment by johnkdevnull** [comment]
As billmeek points out, it is in the LiveTV storage group.  That's because I pressed R to record it while I was watching it on LiveTV.

**Comment by dmfrey** [comment]
Ah, right, missed that.  We are explicitly hiding anything in the live tv
group.

On Thu, Dec 20, 2012 at 12:50 PM, John Kelly notifications@github.comwrote:

> As billmeek points out, it is in the LiveTV storage group. That's because
> I pressed R to record it while I was watching it on LiveTV.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/MythTV-Clients/MythTV-Android-Frontend/issues/94#issuecomment-11583411.

## 

My Google Profile http://www.google.com/profiles/dmfrey

**Comment by johnkdevnull** [comment]
@dmfrey Well can I suggest that you aren't doing that in a manner consistent with Mythfrontend.  My 'Watch Recordings' page isn't set to show LiveTV items but it shows this one because I hit record which I assume makes it a recorded item.  The film shows in both 'All Programmes' and also in 'Watch List'.

**Comment by billmeek** [comment]
I just tested this on my system. I started watching live TV and pressed 'r' to record
the show.

The show did appear on my Recordings page.  I'm on 0.27-pre2 and the latest
Android git.

In both my test and johnkdevnull's, the Storage Group is LiveTV, but the RecGroup is set
to Default. I can see in commit d4d8f6f where the recording is excluded if the RecGroup
is LiveTV, but I can't find that code anymore, unless it was moved to Maven.

@johnkdevnull, what version of MythTV Android Frontend are you running?

**Comment by johnkdevnull** [comment]
@billmeek I'm running version 1.0.2 on a Kindle Fire HD.  It's the current (beta) version available on the Amazon apps store.

If I can download a later version of the apk file from somewhere then let me know...

**Comment by billmeek** [comment]
@johnkdevnull I've only tested 1.0.3 downloads from the Google Play Store, not 
Amazon, but this was just released on the Mythtv users list. It's 1.0.3+

https://www.dropbox.com/s/ju3yw07m0txoy9h/mythtv-for-android-2012-12-19.apk

**Comment by johnkdevnull** [comment]
Ah, ok.  Version 1.0.3 downloaded from the above link now shows my film in the recorded list.  So I was obviously just using an earlier version.
This version now shows each episode three times in the list.  Better too many than too few.  :-) 

This was the first app I looked for when I got a tablet so I'm pretty stoked someone is now building it.  Well done guys!

**Comment by dmfrey** [comment]
John,

That is fix in the repo as well. I am hoping to push this bug fix release
sometime this weekend to the Google play and Amazon app store.

Dan

**Comment by billmeek** [comment]
Marking as closed as original issue is fixed in 1.0.3. Please reopen if you disagree. The 2nd issue
is fixed in 1.0.4.
